spec.emu

```diff
@@ -1164,7 +1164,7 @@ contributors: Luca Casonato, Guy Bedford, Nicolò Ribaudo
               <td>
                 <p><ins>It returns either a normal completion containing the Module Source Object corresponding to this source Module Record's source phase (<emu-xref href="#sec-module-source-objects"></emu-xref>), or a throw completion.</ins></p>
                 <p><ins>When called multiple times on the same Module Record, if GetModuleSource() returns a normal completion it must always return a normal completion containing the same object.</ins></p>
-                <p><ins>The returned object should be an instance of a subclass of %AbstractModuleSource%, and it must have an internal slot [[ModuleSourceClassName]].</ins></p>
+                <p><ins>The returned object should have a [[Prototype]] internal slot whose value is %AbstractModuleSource.prototype%.</ins></p>
```

Comment:
```suggestion
                <p><ins>The returned object must be a Module Source Object.</ins></p>
```

Reply:
Where do we define `Module Source Object` under this framing?

Reply:
This is already stated in the first bullet point, so I think we can skip this change.